src/list/popup/components/item-list-panel.js

```diff
@@ -67,35 +67,46 @@ export default function ItemListPanel({inputRef, noResultsBanner, ...props}) {
     topBorder = "none";
   } else {
     list = <PopupItemList {...props}/>;
+    topBorder = "normal";
 
     if (noResultsBanner) {
-      topBorder = "normal";
       banner = (
         <Localized id="no-results-banner">
-          <PanelBanner border="floating">no rESULTs</PanelBanner>
+          <PanelBanner border="floating" className={styles.panelBanner}>no rESULTs</PanelBanner>
         </Localized>
       );
+    } else if (isFiltering) {
+      const count = props.items.length;
+      banner = (
+          <Localized id="filtered-banner" $count={count}>
+            <PanelBanner border="floating"> {count} eNTRIEs fOUNd</PanelBanner>
+          </Localized>
+      );
     } else {
-      topBorder = "floating";
+      banner = (
```

Comment:
this causes a breakage in `test/unit/list/popup/components/item-list-panel-test.js`.

There is now always a [`PanelBanner`](https://github.com/mozilla-lockbox/lockbox-addon/blob/c36df2c25d65161c72c3eb5021a153879040d876/test/unit/list/popup/components/item-list-panel-test.js#L90), where previously there were some cases it would not be there.